test/uri.jl

```diff
@@ -485,6 +485,9 @@ urltests = URLTest[
         @test HTTP.queryparams(HTTP.URI("http://www.example.com/path/foo+bar/path?query+name=query+value")) ==
             Dict("query name" => "query value")
         @test HTTP.queryparams(HTTP.escapeuri(Dict("a+b" => "c d+e"))) == Dict("a+b" => "c d+e")
+
+        # Issue 540
+        @test HTTP.escapeuri((a=1, b=1)) == "a=1&b=2"
```

Comment:
Looks like you need `(a=1, b=2)` or change the answer to `"a=1&b=1`

Reply:
Oh woops, typo 🙃